### PR TITLE
FEAT: Image attachment

### PR DESCRIPTION
**Pull request by jakubpetrik**

### CODE REVIEW

File: Styles/Classes/Extensions/NSAttributedString+Mutable.swift [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//  Copyright © 2018 Inloop, s.r.o. All rights reserved.
+
+import Foundation
+
+extension NSAttributedString {
+    func mutable() -> NSMutableAttributedString {
+        return mutableCopy() as! NSMutableAttributedString
```
Comment (radimhalfar) [inline Styles/Classes/Extensions/NSAttributedString+Mutable.swift]: How about optional cast ?

File: Styles/Classes/TextStyle.swift [hunk 2]
```diff
@@ -86,25 +89,43 @@ public final class TextStyle: NSObject {
...
 
     public static func +(left: TextStyle, right: TextStyle) -> TextStyle {
         let newAttributes = left.attributes.merging(right.attributes, uniquingKeysWith: { $1 })
-        let newEffects = left.effects.updating(right.effects)
+        let newEffects = left.effects + right.effects
         return TextStyle(attributes: newAttributes, effects: newEffects)
     }
+
+    // MARK:- Updating effects
+    public func appending(_ other: TextEffect...) -> TextStyle {
```
Comment (radimhalfar) [inline Styles/Classes/TextStyle.swift]: Does it filter same TextEffect if exists?

File: Styles/Classes/Extensions/NSAttributedString+Mutable.swift (hunk 1 above)
Comment (jakubpetrik) [reply Styles/Classes/Extensions/NSAttributedString+Mutable.swift]: naah. We know that NSAttributedString's mutable variant is NSMutableAttributedString.

File: Styles/Classes/TextStyle.swift (hunk 2 above)
Comment (jakubpetrik) [reply Styles/Classes/TextStyle.swift]: not right now. Will do.

**Pull request merged**